packages/arch-x86_32/arch-x86_32.1/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "2.0"
+synopsis: "Build OCaml for x86 (32-bit)"
+description: """
+Installing this package causes the OCaml compiler packages to target
+32bit Intel x86 (x86_32/x86).
+
+At present, this facility is only available for the native Windows ports of
+OCaml."""
+maintainer: "David Allsopp <david@tarides.com>"
+authors: "David Allsopp"
+license: "CC0-1.0+"
+homepage: "https://opam.ocaml.org"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+# This package is marked avoid-version in order to steer the default on 64-bit
+# systems towards a 64-bit compiler.
+flags: compiler
+available: os = "win32"
+depends: [
+  ("ocaml-base-compiler" {post & >= "4.13.0~" & <= "5.3.0~beta2"} |
+ "ocaml-variants" {post & >= "4.13.0~" & <= "5.1.1+effect-syntax"})
+  "host-arch-x86_32" {post & = "1"}
+]
+conflict-class: "ocaml-arch"
+x-opam-repository-commit-hash-at-time-of-archiving:
+  "2ac5b4411dc6433623d35bbb1ad092b393d3174e"
+x-reason-for-archiving: ["uninstallable"]
```

Comment:
I find it a bit suprising for this package. How did you compute installability?

Reply:
The tool, archive-opam, looks for `available: false` OR `flags: deprecated` OR `flags: avoid-version`. Thanks for spotting this package, which should be retained (although avoid-version is present). I'll amend the tool to put arch-x86_32 into the allow list -- see dc4d360.